test/cookbooks/cerner_splunk_ingredient_test/recipes/universal_forwarder.rb

```diff
@@ -7,7 +7,7 @@
 splunk_conf 'system/server.conf' do
   config(
     general: {
-      serverName: 'test-forwarder'
+      invalid: 'test-forwarder'
```

Comment:
The actual hostname. I've discovered it's not being overwritten, it's not setting the config in the first place. I think doing `only_if { changed? }` in these sub resources is evaluating changed properties for those sub resources.